CIME/build_scripts/buildlib.spio

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env python3
```

Comment:
Using config_files you can relocate this file to a path within the scorpio repository, that way you can make further changes without the need for a cime PR - I suggest that you move this file.

Reply:
Looks like @jgfouca also agrees with you about the location of the file, let me move the file and then you can review the PR.

Reply:
buildlib.spio is now moved out of CIME to $E3SM_SRC/share/build